src/node_buffer.cc

```diff
@@ -1072,6 +1072,44 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
                                 : -1);
 }
 
+void Swap16(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  THROW_AND_RETURN_UNLESS_BUFFER(env, args.This());
+  SPREAD_ARG(args.This(), ts_obj);
+
+  if (ts_obj_length % sizeof(uint16_t) != 0)
+    return env->ThrowRangeError(
+      "Buffer length must be a multiple of 16-bits");
```

Comment:
nit: wrap multi-line statements in conditionals with `{}`, and double indent with multi-line statements. another thing lint should be catching.